listenbrainz/webserver/static/js/src/RecentListens.tsx

```diff
@@ -564,7 +570,14 @@ export default class RecentListens extends React.Component<
                 <PinnedRecordingCard
                   userName={user.name}
                   pinnedRecording={userPinnedRecording}
+                  className={
+                    this.isCurrentListen(getListenablePin(userPinnedRecording))
+                      ? " current-listen"
+                      : ""
+                  }
                   isCurrentUser={currentUser?.name === user?.name}
+                  playListen={this.playListen}
+                  removePinFromPinsList={() => {}}
```

Comment:
To be implemented in future or forgot to commit?

Reply:
There's no need to pass in a function that does anything since there's no pin list in this component, only in userPins. removePinFromPinsList could also be made into an optional prop but I figured this looks cleaner.